app/models/job_invocation.rb

```diff
@@ -132,7 +132,7 @@ def generate_description!
     description_format.scan(key_re) { |key| input_hash[key.first] }
     self.description = description_format
     input_hash.each do |k, v|
-      self.description.gsub!(Regexp.new("%\{#{k}\}"), v)
+      self.description.gsub!(Regexp.new("%\{#{k}\}"), v || '')
```

Comment:
`.gsub!` must be passed a string, and when working outside the UI (API, tests, etc), v can be nil.